.readme-partials.yaml

```diff
@@ -22,9 +22,9 @@ custom_content: |
 
   TopicPath topicPath =
       TopicPaths.newBuilder()
-          .setProject(ProjectNumber.of(projectNumber))
```

Comment:
Looks like the changes in this commit are just undoing things from the previous one. Can you squash the commits and force push your history?

Reply:
Commits will be squashed by github. I'm not going to squash them here.